projects/canopy/src/lib/paginator/paginator.component.html

```diff
@@ -0,0 +1,52 @@
+<lg-select-field class="lg-paginator__page-size" *ngIf="!hidePageSize">
+  <span class="lg-paginator__page-size-label">Items per page</span>
+
+  <select lgSelect [ngModel]="pageSize" (ngModelChange)="handleSelectPageSize($event)">
+    <option *ngFor="let size of pageSizes" [value]="size">{{ size }}</option>
+  </select>
+</lg-select-field>
+
+<span class="lg-paginator__status"
+  >{{ currentPageStart }} - {{ currentPageEnd }} of {{ length }}</span
+>
+
+<div class="lg-paginator__controls">
```

Comment:
From an accessibility point of you I think it would be good for this to be marked as a `navigation` and to have the buttons within a list.
Also, one comment about `aira-label` is that some screen readers don't read it so it would be better to have the label of the button within the button e.g.: `<button ...><span class="lg-visually-hidden">Show first page</span></button>`.

I think this is a good article that explains how to make a paginator accessible: https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-pagination

I've noticed this version has only the "arrows" and no page numbers? should we add the capability of having numbers too? e.g.:

<img width="470" alt="Screenshot 2021-01-19 at 17 16 46" src="https://user-images.githubusercontent.com/8397116/105069667-2cdb1f80-5a7a-11eb-941e-831389643f14.png">

Reply:
Thanks for that, I'll have a read.

Regarding page numbers and how it works I just modeled the component on the basic functionality for mat-paginator. I don't have much of an opinion on how it should function so open to design input.

Reply:
@alexcanning any opinion on this?